e2e/common/createCitizenUser/createCitizenUser.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  initializeAPIContext,
+  getAccessToken,
+  createCitizenUser,
+} from "./idamApiHelper";
+
+/**
+ * Sets up a user by getting an access token and creating a citizen user.
+ * @returns {Promise<{ email: string; password: string; id: string }>} User information if successful
+ * @throws Will throw an error if any step fails
+ */
+export async function setupUser(): Promise<{
```

Comment:
Does IDAM also provide a way to cleanup these users?

Reply:
From (https://tools.hmcts.net/confluence/display/SISM/Creating+Test+Users+in+IDAM): "Test users created through idam-testing-support-api are automatically cleared down after a period of time. Note that the clear down process uses the bearer token as its key, so all users created using the same token will be cleared down as one batch." 

but actually i think i should be using the same bearer token for each set of tests at least